File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/inventory/services/ReservationService.java
package com.metaengine.demo.inventory.services;

import java.util.List;

import com.metaengine.demo.inventory.aggregates.StockItem;

/** ReservationService domain service. */
public class ReservationService {

    /** Reserve stock and return the resulting StockItem. */
    public StockItem create(StockItem input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a StockItem by id (may return null). */
    public StockItem findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List stock items up to limit. */
    public List<StockItem> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a StockItem by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-003/a-mcp/output/inventory/services/reservation_service.py
from inventory.aggregates.stock_item import StockItem

"""ReservationService service."""
class ReservationService:
    def create(self, input: StockItem) -> StockItem:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> StockItem | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[StockItem]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

